src/client/components/network-editor/index.js

```diff
@@ -53,7 +53,10 @@ export class NetworkEditor extends Component {
       },
       {
         selector: 'edge',
-        style: this.cy.vizmapper().edgeStyleBlock()
+        style: { 
+          ...this.cy.vizmapper().edgeStyleBlock(), 
+          'curve-style': 'bezier',
```

Comment:
In future, the `curve-style` could vary depending on the graph and style that we have, i.e.:

- Multigraph: bezier by default
- Simple graph: haystack by default
- Explicit haystack style set: haystack